download/bindings-fr.php

```diff
@@ -55,7 +55,7 @@
         <tr>
             <th rowspan="1">Crystal</th>
             <td><a href="https://github.com/oprypin/crsfml" class="website"><img src="../images/bindings/crsfml.png" alt="CrSFML"/></a></td>
-            <td>2.5</td>
```

Comment:
Why still bother mentioning anything but 2.6?

Reply:
Not sure. The binding right below mentions `2.0 - 2.5`

Reply:
Then maybe all occurrences should be removed